prov/efa/src/rxr/rxr_read.c

```diff
@@ -516,22 +516,15 @@ int rxr_read_post_once(struct rxr_ep *ep, struct rxr_read_entry *read_entry,
 		       void *local_buf, size_t len, void *desc,
 		       uint64_t remote_buf, size_t remote_key)
 {
-	struct fi_msg_rma msg = {0};
-	struct iovec iov;
-	struct fi_rma_iov rma_iov;
 	struct rxr_pkt_entry *pkt_entry;
 	struct rdm_peer *peer;
 	struct efa_ep *efa_ep;
+	struct efa_qp *qp;
+	struct efa_conn *conn;
+	struct ibv_sge sge_list[1];
```

Comment:
looks like you just need 1 `sge`  instead of an array

Reply:
fixed